### PR TITLE
new workflows: Deploy Patch with BigFix from Slack and Teams

### PR DESCRIPTION
**Pull request by sengleson-r7**
New workflows to deploy a patch from BigFix from Slack or Teams.

### CODE REVIEW

File: workflows/Deploy_Patch_With_BigFix_From_Teams/workflow.spec.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [remediation_management]
+  keywords: [work_from_home, bigfix, patch_management, microsoft_teams, chat]
+  features: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/my_cool_workflow
```
Comment (zyoutz-r7) [inline workflows/Deploy_Patch_With_BigFix_From_Teams/workflow.spec.yaml:15]: Update

File: workflows/Deploy_Patch_With_BigFix_From_Slack/workflow.spec.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [remediation_management]
+  keywords: [work_from_home, bigfix, patch_management, slack, chat]
+  features: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/my_cool_workflow
```
Comment (zyoutz-r7) [inline workflows/Deploy_Patch_With_BigFix_From_Slack/workflow.spec.yaml:15]: Update

File: workflows/Deploy_Patch_With_BigFix_From_Teams/workflow.spec.yaml (hunk 1 above)
Comment (sengleson-r7) [reply workflows/Deploy_Patch_With_BigFix_From_Teams/workflow.spec.yaml]: fixed!

File: workflows/Deploy_Patch_With_BigFix_From_Slack/workflow.spec.yaml (hunk 2 above)
Comment (sengleson-r7) [reply workflows/Deploy_Patch_With_BigFix_From_Slack/workflow.spec.yaml]: fixed!

**Pull request merged**